internal/cmdrpkgget/command.go

```diff
@@ -74,14 +80,29 @@ type runner struct {
 	Command *cobra.Command
 
 	// Flags
-	name       string
-	revision   string
-	printFlags *get.PrintFlags
+	packageName   string
+	revision      string
+	printFlags    *get.PrintFlags
+	AllNamespaces bool
 
 	requestTable bool
 }
 
 func (r *runner) preRunE(cmd *cobra.Command, args []string) error {
+	if *r.cfg.Namespace == "" {
```

Comment:
Could we handle this in a single place for all the rpkg commands?

Reply:
Good idea - I'll look at refactoring it out!

Reply:
Done in #3541